_pubs/explanation-alignment.md

```diff
@@ -0,0 +1,21 @@
+---
+title: 'Explanation Alignment: Quantifying the Correctness of Model Reasoning At Scale'
+authors:
+  - key: hbang
+  - key: aboggust
+  - key: arvindsatya
+venue: eXCV
+year: 2024
+date: 2024-09-29
+tags:
```

Comment:
Thanks @hhybang! Could we also add a tag for `quantitative methods`?

Reply:
Oh and could we add a top-level YAML key-pair of `type: workshop` to indicate that this is a workshop paper.

Reply:
Yes!